<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogNetworkCreate</class>
 <widget class="QDialog" name="DialogNetworkCreate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>549</width>
    <height>294</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushUp">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>50</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushDown">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>90</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushRemove">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>20</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushAdd">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>20</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableNetwork">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>471</width>
     <height>201</height>
    </rect>
   </property>
   <property name="rowCount">
    <number>0</number>
   </property>
   <property name="columnCount">
    <number>3</number>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>150</number>
   </attribute>
   <column>
    <property name="text">
     <string>Layer Type</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Activation Type</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Nodes</string>
    </property>
   </column>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>260</y>
     <width>166</width>
     <height>21</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="textTicker">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>20</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhUppercaseOnly</set>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="tabChangesFocus">
    <bool>true</bool>
   </property>
   <property name="plainText">
    <string/>
   </property>
   <property name="overwriteMode">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string>XYZ</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>47</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Ticker:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
